lib/rouge/lexers/shell.rb

```diff
@@ -9,15 +9,17 @@ class Shell < RegexLexer
 
       tag 'shell'
       aliases 'bash', 'zsh', 'ksh', 'sh'
-      filenames '*.sh', '*.bash', '*.zsh', '*.ksh', '.bashrc', '.zshrc',
-                '.kshrc', '.profile', 'APKBUILD', 'PKGBUILD', '*.ebuild',
+      filenames '*.sh', '*.bash', '*.zsh', '*.ksh', '.bashrc', '.zshrc', 'zshrc',
+                '.kshrc', '.profile', '.zshenv', 'zshenv', '.zprofile', 'zprofile',
+                'APKBUILD', 'PKGBUILD', '*.ebuild',
```

Comment:
**note:** Additional zsh files can be found at https://zsh.sourceforge.io/Doc/Release/Files.html#Files-1